src/cloudant/database.py

```diff
@@ -329,16 +329,21 @@ def get_view_result(self, ddoc_id, view_name, raw_result=False, **kwargs):
         else:
             return view.result
 
-    def create(self):
+    def create(self, **kwargs):
         """
         Creates a database defined by the current database object, if it
         does not already exist and raises a CloudantException if the operation
         fails.  If the database already exists then this method call is a no-op.
 
+        :param bool none_on_exists: Boolean flag dictating whether or
+            not to return none object when the database already exists.
         :returns: The database object
         """
         if self.exists():
-            return self
+            if kwargs.get('none_on_exists', True):
+                return None
+            else:
+                return self
```

Comment:
Seems a good idea to me. I will rewrite and submit it again. Thanks for the idea.